docs/MO_DG/prepare_model/convert_model/Convert_Model_From_Caffe.md

```diff
@@ -80,7 +80,7 @@ CLI Examples Using Caffe-Specific Parameters
 Custom Layer Definition
 ########################
 
-Internally, when you run Model Optimizer, it loads the model, goes through the topology, and tries to find each layer type in a list of known layers. Custom layers are layers that are not included in the list. If your topology contains such kind of layers, Model Optimizer classifies them as custom.
+Internally, when you run Mdel Conversion API, it loads the model, goes through the topology, and tries to find each layer type in a list of known layers. Custom layers are layers that are not included in the list. If your topology contains such kind of layers, Model Conversion API classifies them as custom.
```

Comment:
This is applicable for all FW formats, not only for Caffe. Should be moved to the common part.
```suggestion
Internally, when you run Model Conversion API, it loads the model, goes through the topology, and tries to find each layer type in a list of known layers. Custom layers are layers that are not included in the list. If your topology contains such kind of layers, Model Conversion API classifies them as custom.
```

Reply:
> This is applicable for all FW formats, not only for Caffe. Should be moved to the common part.

Have moved the text with the [0d56b4c](https://github.com/openvinotoolkit/openvino/pull/17454/commits/0d56b4c875f19456d09c623fe5f039ca0e297899) commit.